curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-36-double-base-palindromes.english.md

```diff
@@ -7,9 +7,13 @@ forumTopicId: 302020
 
 ## Description
 <section id='description'>
-The decimal number, 585 = 10010010012 (binary), is palindromic in both bases.
-Find the sum of all numbers, less than n, whereas 1000 <= n <= 1000000, which are palindromic in base 10 and base 2.
+
+The decimal number, 585 = 1001001001<sub>2</sub> (binary), is palindromic in both bases.
+
+Find the sum of all numbers, less than n, whereas 1000 ≤ `n` ≤ 1000000, which are palindromic in base 10 and base 2.
```

Comment:
I agree, it would be better to be consistent wherever possible.

I'll also go back and change the `<code>` tags in the description to backticks wherever possible.